frontend/src/stores/knowledges/actions.ts

```diff
@@ -20,5 +21,10 @@ async function saveKnowledge(knowledge: Knowledge): Promise<void> {
 function setKnowledges(newKnowledges: Knowledge[]): void {
   knowledges.value = newKnowledges
 }
+function removeKnowledge(knowledgeId: string) {
```

Comment:
```suggestion
function removeKnowledge(knowledgeId: number) {
```

An ID is a number, not a string